semgrep/javascript/nodejs/express/nodejs-express-mysql-injection.yml

```diff
@@ -0,0 +1,32 @@
+rules:
+  - id: nodejs-mysql-body-inection 
+    languages:
+      - javascript
+    message: | 
+      Potential SQL Injection identified. Consider using the connection.escape method built in node-mysql
+    patterns:
+      - pattern-inside:  |
+          $MYSQL=require('mysql')
```

Comment:
there are other require/import styles (can look at cookie attribute rules for reference) that can be used in JS/TS. We should probably include all styles or just not include the "require" in the rule.